plugins/modules/elasticsearch_plugin.py

```diff
@@ -163,33 +163,36 @@ def parse_error(string):
 
 
 def install_plugin(module, plugin_bin, plugin_name, version, src, url, proxy_host, proxy_port, timeout, force):
-    cmd_args = [plugin_bin, PACKAGE_STATE_MAP["present"]]
+    cmd = [plugin_bin, PACKAGE_STATE_MAP["present"]]
     is_old_command = (os.path.basename(plugin_bin) == 'plugin')
 
     # Timeout and version are only valid for plugin, not elasticsearch-plugin
     if is_old_command:
         if timeout:
-            cmd_args.append("--timeout %s" % timeout)
+            cmd.append("--timeout %s" % timeout)
```

Comment:
This should be two new list elements, like
```suggestion
            cmd.append("--timeout")
            cmd.append("%s" % timeout)
```

Reply:
Hi! Good suggestion, I applied your suggestion.